src/lib/integration_test_lib/wait_condition.ml

```diff
@@ -50,8 +50,8 @@ struct
     in
     { description= Printf.sprintf "\"%s\" to initialize" (Node.id node)
     ; predicate= Network_state_predicate (check (), check)
-    ; soft_timeout= Literal (Time.Span.of_min 5.0)
-    ; hard_timeout= Literal (Time.Span.of_min 7.5) }
+    ; soft_timeout= Literal (Time.Span.of_min 10.0)
+    ; hard_timeout= Literal (Time.Span.of_min 15.0) }
```

Comment:
is this intentional?

Reply:
Yes. I was not able to reliably get nodes to start in 7.5 mins anymore (they would initialize right after the hard timeout in most cases). I want to look into decreasing these timeouts long term still.